test/e2e_env/universal/envoyconfig/testdata/sidecars/meshfaultinjection/inbound-rules.input.yaml

```diff
@@ -0,0 +1,24 @@
+type: MeshFaultInjection
+name: mfi-1
+mesh: envoyconfig
+labels:
+  kuma.io/effect: shadow
+spec:
+  targetRef:
+    kind: Mesh
+  rules:
+    - matches:
+        - spiffeID:
+            type: Prefix
+            value: spiffe://default.default.mesh.local
+      default:
+        http:
+          - abort:
+              httpStatus: 500
+              percentage: '2.5'
+          - abort:
+              httpStatus: 500
+              percentage: 10
```

Comment:
why two aborts? 🤔

Reply:
for each of these we create separate listener, I wanted to have a more complex example in docs